src/settings.ts

```diff
@@ -585,6 +585,7 @@ export const POSTERPAGE_MAX_VIDEO_PARTICIPANTS = 10;
 export const SEARCH_DEBOUNCE_TIME = 200; // ms
 
 export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 48;
```

Comment:
Please remove the old `const` if it's not being used anymore.

Reply:
I expect this is still being used in the `PosterHall`

Reply:
@denisdimitrov that const is used in PosterHall